Server/src/services/validationSchema.ts

```diff
@@ -0,0 +1,128 @@
+//Running and installing [npm i jsonschema] is required for this page to work
+//More details and examples about json schema validator can be found at:
+// https://github.com/tdegrunt/jsonschema#readme
+
+
+export const schemaValidator = {
+    'baseSchema': {
+        "id": "/baseSchema",
+        "type": "object",
+        "properties": {
+            "reference": {
+                "$ref": "/referenceValidationSchema"
+            },
+            "datasetName": { "type": "string" },
+            "material": {
+                "$ref": "/materialValidationSchema"
+            },
+            "category": { "type": "string" },
+            "subcategory": { "type": "string" },
+            "data": {
+                "$ref": "/dataValidationSchema"
+            }
+        },
+        "required": ["reference", "datasetName", "material", "data"]
+    },
+
+
+    'referenceValidationSchema': {
+        "id": "/referenceValidationSchema",
+        "type": "object",
+        "properties": {
+            "type": { "type": "string" },
+            "publisher": { "type": "string" },
+            "authors": {
+                "$ref": "/authorsValidationSchema"
+            },
+            "title": { "type": "string" },
+            "volume": { "type": "integer" },
+            "pages": { "type": "integer" },
+            "year": { "type": "integer" }
+        },
+        "required": ["authors"]
+    },
+
+    'authorsValidationSchema': {
```

Comment:
maybe for the future, im sure there is a nice way to achieve this with react instead of hardcoding schema, our models may change, and then this schema too, but we will forget to change it and it will lead to a bug

Reply:
comments have been removed from the files

Reply:
the schema was based on the contents of the json example file that we have been using for the file upload testing. will look into optimize the schemas for future sprints